src/alitra/transformations.py

```diff
@@ -0,0 +1,124 @@
+from typing import Union
+
+import numpy as np
+from scipy.spatial.transform import Rotation
+
+from .models.frame import Frame
+from .models.orientation import Orientation
+from .models.pose import Pose
+from .models.position import Position, Positions
+from .models.transform import Transform
+
+
+def transform_position(
```

Comment:
Is there a reason why these are standalone functions and not just class methods for Transform?

Reply:
I guess the same could be said for `align_positions` and `align_maps` aswell. They could be `@staticmethod` on the `Transform` class. I though maybe keeping the classes a bit smaller and simpler would make it more readable. Any thoughts?

Reply:
Transforms moved to Transform class